source/common/runtime/runtime_features.cc

```diff
@@ -1,26 +1,34 @@
 #include "source/common/runtime/runtime_features.h"
 
+#include "absl/flags/commandlineflag.h"
 #include "absl/flags/flag.h"
 #include "absl/strings/match.h"
 #include "absl/strings/str_replace.h"
 
-/* To add a runtime guard to Envoy, add 2 lines to this file.
- *
- * RUNTIME_GUARD(envoy_reloadable_features_flag_name)
- *
- * to the sorted macro block below and
- *
- * &FLAGS_envoy_reloadable_features_flag_name
- *
- * to the runtime features constexpr.
- *
- * The runtime guard to use in source and release notes will then be of the form
- * "envoy.reloadable_features.flag_name" due to the prior naming scheme and swapPrefix.
- **/
-
 #define RUNTIME_GUARD(name) ABSL_FLAG(bool, name, true, "");        // NOLINT
 #define FALSE_RUNTIME_GUARD(name) ABSL_FLAG(bool, name, false, ""); // NOLINT
 
+// Add additional features here to enable the new code paths by default.
+//
+// Per documentation in CONTRIBUTING.md is expected that new high risk code paths be guarded
+// by runtime feature guards. If you add a guard of the form
+// RUNTIME_GUARD(envoy_reloadable_features_my_feature_name)
+// here you can guard code checking against "envoy.reloadable_features.my_feature_name".
+// Please note the swap of envoy_reloadable_features_ to envoy.reloadable_features.!
```

Comment:
What happens if people mess this up? Will it error out? If not, can we do something to avoid this mismatch?

Reply:
it'll work just fine actually, it just won't have standard envoy guard naming.